addons/sairam.quadtree/QuadTree.gd

```diff
@@ -129,7 +140,7 @@ func clear() -> void:
 	if !_is_leaf:  # if the self is not leaf
 		for child in _children:
 			child.clear()  # clear all it's children
-		_is_leaf = true
+		_set_as_empty_leaf()
```

Comment:
Why was this moved to a function?

Reply:
It needed to be done in three places, and what was done wasn't trivial - it needed to both set is_leaf *as well as* set all children to null. It seemed to be enough of a concept to warrant its own function. It's more than just setting is_leaf now. That was part of the unsubdivide problem - _children wasn't being set back to all null.